docs/hosting/by-jina.md

```diff
@@ -0,0 +1,95 @@
+# Hosted by Jina AI
```

Comment:
What's the difference betwee *hostted by Jina AI* and *hosted by jCloud*? I think this title is misleading. Maybe we can just use **# Cloud Hosting**?

Reply:
I think this page would explain the SaaS product.

Reply:
One is hosted **by** Jina, right now the user can only access it but not control it.
The other is hosted **on** Jcloud, the user has full control of it

Reply:
But this title does not tell the difference